### DISCUSSION TITLE
samples/wls-k8s-domain is removed

### DISCUSSION THREAD

**Discussion by ssarraju** [oracle/docker-images#2586] [General]
Team, I was following this [blog](https://medium.com/@brunoborges/weblogic-on-kubernetes-give-it-a-try-795cf9458e33) and sample sometime ago and I don't see this folder anymore and it seems to be deleted. Any idea why? 

https://github.com/oracle/docker-images/tree/eedfa96e8e0ab14a128c1fd7ac2b64f7cf075411/OracleWebLogic/samples/wls-k8s-domain

**Comment by rjeberhard** [comment]
Our current solution is based on a Kubernetes operator for WebLogic: https://github.com/oracle/weblogic-kubernetes-operator. 

We were no longer maintaining and testing this sample so we removed it.
